packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -177,6 +184,7 @@ class _InspectorTreeState extends State<InspectorTree>
   Rect currentAnimateTarget;
 
   AnimationController constraintDisplayController;
+  FocusNode focusNode = FocusNode();
```

Comment:
make final.

Reply:
It can't be final now because I'm initializing it in initState()